pyproject.toml

```
@@ -21,7 +21,7 @@ main = [
     "validators>=0.34",
 ]
 pre-commit = ["pre-commit>=4.0"]
-test = ["pytest>=8.3"]
+test = ["gitpython>=3.1.44", "pytest-cov>=6.1.1", "pytest>=8.3"]
```

Specific patch version of dependencies that adhere to semantic versioning is not necessary.
```suggestion
test = ["gitpython>=3.1.44", "pytest-cov>=6.1", "pytest>=8.3"]
```